Can't resolve 'raw-loader'

As of 0.28, I'm getting this error when doing a webpack build:

```
ERROR in /Users/grant/Sites/jupyter/jupyter-renderers/packages/plotly-extension/lib/index.js
Module not found: Error: Can't resolve 'raw-loader' in '/Users/grant/Sites/jupyter/jupyter-renderers/packages/plotly-extension/lib'
 @ /Users/grant/Sites/jupyter/jupyter-renderers/packages/plotly-extension/lib/index.js 30:22-72
 @ ./build/index.out.js
```

And this error in the browser:

```
index.js:220 Error: Cannot find module "raw-loader!plotly.js/dist/plotly.min.js"
    at webpackMissingModule (index.js:30)
    at Object.loadPlotly (index.js:30)
    at new RenderedPlotly (index.js:58)
    at Object.createRenderer (index.js:108)
    at RenderMime.createRenderer (rendermime.js:122)
    at OutputArea._createOutputItem (widget.js:349)
    at OutputArea._insertOutput (widget.js:332)
    at new OutputArea (widget.js:100)
    at new CodeCell (widget.js:414)
    at ContentFactory.createCodeCell (widget.js:462)
```

--- a/jupyterlab/webpack.config.js
+++ b/jupyterlab/webpack.config.js
@@ -44,6 +44,13 @@ Object.keys(jlab.linkedPackages).forEach(function (name) {
   linkedPackages[name] = fs.realpathSync(jlab.linkedPackages[name]);
 });
 
+// The valid module paths.
+var modules = [
+  path.resolve(__dirname, './node_modules'),
+  path.resolve(__dirname, '../node_modules')
+];
+
+
 module.exports = {
   entry:  path.resolve(buildDir, 'index.out.js'),
   output: {
@@ -70,8 +77,10 @@ module.exports = {
   },
   resolve: {
     alias: linkedPackages,
-    modules: [path.resolve(__dirname, './node_modules'),
-              path.resolve(__dirname, '../node_modules')]
+    modules: modules
+  },
+  resolveLoader: {
+    modules: modules
   },
   watchOptions: {
     ignored: /node_modules/

